htdocs/web_portal/views/scopes.php

```diff
@@ -1,6 +1,75 @@
 <div class="rightPageContainer">
-    <h1>Resource Scoping</h1>
-    <br />
+
+    <div style="float: left;">
+        <h1 style="float: left; margin-left: 0em;">
+            Scopes
+        </h1>
+        <span style="clear: both; float: left; padding-bottom: 0.4em;">
+            Click on the name of a scope to <?=$params['UserIsAdmin'] ? "edit it and " : "";?> view objects with that scope tag.
+        </span>
+    </div>
+    <!-- hide add when read only or user is not admin -->
+    <?php if(!$params['portalIsReadOnly'] && $params['UserIsAdmin']):?>
+        <div style="float: right;">
+            <center>
+                <a href="index.php?Page_Type=Admin_Add_Scope">
+                <img src="<?php echo \GocContextPath::getPath()?>img/add.png" height="25px" />
+                <br />
+                <span>Add Scope</span>
+                </a>
+            </center>
+        </div>
+    <?php endif; ?>
+
+    <?php $numberOfScopes = count($params['Scopes'])?>
+    <div class="listContainer">
+        <span class="header listHeader">
+            <?php echo $numberOfScopes ?> Scope<?php if($numberOfScopes) echo "s"?>
```

Comment:
```suggestion
            <?php echo $numberOfScopes ?> Scope<?php if ($numberOfScopes  != 1) echo "s"?>
```

Reply:
Yep, but is `!==` not generally preferable?

Reply:
I guess we could write a book on that one 😀 It's not an operator I'm in the habit of using but you're probably correct ... generally.

Reply:
`==` is loose comparison, `===` is strict comparisons. For simple types (int, string, bool) I would use strict comparisons because it seems you can get into a situation where  `true` is loosely `false` at least under older versions of php.

https://www.php.net/manual/en/types.comparisons.php
https://news.ycombinator.com/item?id=9485855